docs/sources/administration/manage-users-and-permissions/_index.md

```diff
@@ -1,6 +1,6 @@
 +++
+aliases = ["/docs/grafana/latest/administration/manage-users-and-permissions/", "/docs/grafana/latest/administration/manage-users-and-permissions"]
```

Comment:
Duplicates save for the trailing slash.

Reply:
Great catch. I've updated the tool to use a canonical form that includes a trailing slash and matches the URL path as seen in the browser. This instance and the others are fixed in aa55c56d5f.